src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.tsx

```diff
@@ -16,9 +16,9 @@ function AnnotationsTargetedTooltip({ children, isEnabled = false }: Props): JSX
     return (
         <TargetedClickThroughTooltip
             className="bp-AnnotationsTooltip"
+            closeOnClickOutside
             shouldTarget={isEnabled}
             showCloseButton
-            tetherElementClassName="bp-AnnotationsTooltipTether"
```

Comment:
Just to confirm: Is this change intentional? Is the increased z-index no longer needed?

Reply:
@jstoffan - Correct, we no longer need the z-index, because if we click on the share button which opens the modal the tooltip will now close.